### PR TITLE
{bio}[foss/2023a] lil-aretomo v0.1.1

### PR DESCRIPTION
**Pull request by ThomasHoffmann77**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/c99fa72e9cb09a992760d2487def9602 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen3

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/17df587d76480889b5c973d0da214c61 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen3

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/lil-aretomo/lil-aretomo-0.1.1-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+version = '0.1.1'
+
+homepage = 'https://github.com/teamtomo/lil-aretomo'
+description = """
+A lightweight Python API for AreTomo.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/l/lil-aretomo/lil-aretomo-0.1.1-foss-2023a.eb:14]:
```suggestion
```

File: easybuild/easyconfigs/l/lil-aretomo/lil-aretomo-0.1.1-foss-2023a.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    ('lil_aretomo', version, {
+        'checksums': ['02ccb0efbf2c06304570117f142e78331bfffdde46864e22de11c6cd7f30f178'],
+    }),
+]
+
+sanity_pip_check = True
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/'],
+}
```
Comment (boegel) [inline easybuild/easyconfigs/l/lil-aretomo/lil-aretomo-0.1.1-foss-2023a.eb:36]: This is the default when using `PythonBundle` easyblock, so can be removed
```suggestion
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3117.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/c309fee6005597a8a3d423401c60aaae for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ThomasHoffmann77!

**Pull request merged**